bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
```

Comment:
As far as I can tell, the init script for debian does indeed use this script (with the hardcoded path) for tasks (https://github.com/theforeman/foreman-tasks/blob/master/deploy/foreman-tasks.init)